### ISSUE TITLE
[Quest][Classic] Spawn issue in quest "Investigate the Blue Recluse"

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5417]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This is a mage quest. No matter where I use the cantation of manifestation 4 Rift Spawns are summoned making it impossible for even a lvl 20 mage to complete the quest when it's a lvl 15/16 quest.

**Expected behaviour**: Tell us what should happen instead
By right, there should only be one Rift Spawn spawned when the cantation of manifestation is used in the cellar/fireplace/under the stairs.

**Steps to reproduce the problem**:

1. Go to the cellar/fireplace/under the stairs and use the cantation of manifestation
2. Observe the number of Rift Spawn spawned
3. Feel free to try the summoning in other areas in Blue Recluse

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/39165345-93d8a0ec-47b6-11e8-93f8-1e8b3bdf3c6f.png)
you can see in the chatbox that 4 Rift Spawns are spawned.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/
http://www.wowhead.com/quest=1920/investigate-the-blue-recluse
**Include Screenshots from the issue if necessary**

**Comment by wrathofthebug** [comment]
Despite the "full rework of quest" stated in the latest changelog, this quest is still not working as intended and the same problems and issues remain.

**Comment by xrraynor** [comment]
I can confirm that this is still an issue on the Horde version of the quest "Investigate the Alchemy Shop" as of February 11, 2020. Very frustrating as I have spent over an hour trying to complete it using various tips from various forums but as this post shows the Rift Spawn will not get stunned.

**Comment by ashphoenix92** [comment]
Still doesn't work. Tried it recently (March, 2020). Rifts are spawned in the Blue Recluse, but they don't get stunned. Tried killing them and placing the trap - still nothing. Tried placing the trap and then killing them - still no use. I've tried different approaches - none has worked.
